filters/fasttext_filter.json

```diff
@@ -0,0 +1,389 @@
+                "gu",
+                "gv",
+                "he",
+                "hi",
+                "hif",
+                "hr",
+                "hsb",
+                "ht",
+                "hu",
+                "hy",
+                "ia",
+                "id",
+                "ie",
+                "ilo",
+                "io",
+                "is",
+                "it",
+                "ja",
+                "jbo",
+                "jv",
+                "ka",
+                "kk",
+                "km",
+                "kn",
+                "ko",
+                "krc",
+                "ku",
+                "kv",
+                "kw",
+                "ky",
+                "la",
+                "lb",
+                "lez",
+                "li",
+                "lmo",
+                "lo",
+                "lrc",
+                "lt",
+                "lv",
+                "mai",
+                "mg",
+                "mhr",
+                "min",
+                "mk",
+                "ml",
+                "mn",
+                "mr",
+                "mrj",
+                "ms",
+                "mt",
+                "mwl",
+                "my",
+                "myv",
+                "mzn",
+                "nah",
+                "nap",
+                "nds",
+                "ne",
+                "new",
+                "nl",
+                "nn",
+                "no",
+                "oc",
+                "or",
+                "os",
+                "pa",
+                "pam",
+                "pfl",
+                "pl",
+                "pms",
+                "pnb",
+                "ps",
+                "pt",
+                "qu",
+                "rm",
+                "ro",
+                "ru",
+                "rue",
+                "sa",
+                "sah",
+                "sc",
+                "scn",
+                "sco",
+                "sd",
+                "sh",
+                "si",
+                "sk",
+                "sl",
+                "so",
+                "sq",
+                "sr",
+                "su",
+                "sv",
+                "sw",
+                "ta",
+                "te",
+                "tg",
+                "th",
+                "tk",
+                "tl",
+                "tr",
+                "tt",
+                "tyv",
+                "ug",
+                "uk",
+                "ur",
+                "uz",
+                "vec",
+                "vep",
+                "vi",
+                "vls",
+                "vo",
+                "wa",
+                "war",
+                "wuu",
+                "xal",
+                "xmf",
+                "yi",
+                "yo",
+                "yue",
+                "zh"
+            ]
+            
+        },
+        "LANG2":
+        {
+            "type": "str",
+            "default": "en",
+            "allowed_values":
+            [        
```

Comment:
Why are some of the languages dual letter code and some single letter? It's a semi disjoint list it seems where some are duplicated and some aren't.

Reply:
Well I took the list from the fasttext page: https://fasttext.cc/docs/en/language-identification.html
They say it's a list of ISO codes :man_shrugging: